go/libkb/upak_lite.go

```diff
@@ -0,0 +1,359 @@
+package libkb
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/buger/jsonparser"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func LoadUPAKLite(arg LoadUserArg) (ret *keybase1.UpkLiteV1AllIncarnations, err error) {
```

Comment:
maybe this should be private function if you expect to only call it through a regular CachedUPAKLoader